internal/host/validator.go

```diff
@@ -490,6 +490,8 @@ func (v *validator) diskEncryptionRequirementsSatisfied(c *validationContext) (V
 		}
 		return status, fmt.Sprintf("The host's TPM version is not supported, expected-version: %s, actual-version: %s",
 			models.InventoryTpmVersionNr20, c.inventory.TpmVersion)
+	case ValidationPending:
+		return status, "disk encryption check was not performed yet"
```

Comment:
I don't understand it well enough, please confirm that this message makes sense for both day-1 and day-2 - as both of them can result in returning Pending

Reply:
Fair point.
It **should** use the same code path, and the msg should make sense for day 2.
However, this will get covered and validated with [MGMT-13461](https://issues.redhat.com/browse/MGMT-13461), which focuses on day 2.